### ISSUE TITLE
Addition of management interfaces

### ISSUE THREAD

**Issue by imduffy15** [awslabs/aws-transit-vpc#3]
It would be neat to see a secondary interface attached for management purposes so you can make configuration changes without the fear of locking yourself out and having to restart the CSR device.

**Comment by stevemorad** [comment]
Would you provide more information about what, besides adding a secondary ENI to the CloudFormation template, you'd like to see for the management interface? Any specific configuration that you are looking for?

**Comment by imduffy15** [comment]
Hi @stevemorad 

We ended up doing this on a very customized version of this solution.

We wanted an interface that we could connect to the instance with over SSH and modify the interface the VPNs connect in via without possibly locking ourselves out of the box.

Ended up with the following additions on the userdata:

```
ios-config-27="ip vrf management"
ios-config-28="interface GigabitEthernet2"
ios-config-29="ip address dhcp"
ios-config-30="interface GigabitEthernet1"
ios-config-31="ip vrf forwarding management"
ios-config-32="ip address dhcp"
ios-config-33="ip route vrf management 0.0.0.0 0.0.0.0 172.28.128.33"
```

where `172.28.128.33` was automatically generated via my provisioning script to find the subnet router. (I'd imagine there's better ways of doing this but my cisco knowledge is non-existent)
